go/oasis-node/cmd/debug/txsource/workload/parallel.go

```diff
@@ -0,0 +1,119 @@
+package workload
+
+import (
+	"context"
+	"fmt"
+	"math/rand"
+	"sync"
+	"time"
+
+	"google.golang.org/grpc"
+
+	"github.com/oasislabs/oasis-core/go/common/crypto/signature"
+	memorySigner "github.com/oasislabs/oasis-core/go/common/crypto/signature/signers/memory"
+	"github.com/oasislabs/oasis-core/go/common/logging"
+	consensus "github.com/oasislabs/oasis-core/go/consensus/api"
+	"github.com/oasislabs/oasis-core/go/consensus/api/transaction"
+	runtimeClient "github.com/oasislabs/oasis-core/go/runtime/client/api"
+	staking "github.com/oasislabs/oasis-core/go/staking/api"
+)
+
+const (
+	NameParallel = "parallel"
+
+	parallelSendWaitTimeoutInterval = 30 * time.Second
+	parallelSendTimeoutInterval     = 60 * time.Second
+	parallelConcurency              = 200
+	parallelTxGasAmount             = 10
+)
+
+var parallelLogger = logging.GetLogger("cmd/txsource/workload/parallel")
+
+type parallel struct{}
+
+func (parallel) Run(gracefulExit context.Context, rng *rand.Rand, conn *grpc.ClientConn, cnsc consensus.ClientBackend, rtc runtimeClient.RuntimeClient) error {
+	ctx := context.Background()
+
+	accounts := make([]signature.Signer, parallelConcurency)
+	var err error
+	fac := memorySigner.NewFactory()
+	for i := range accounts {
+		accounts[i], err = fac.Generate(signature.SignerEntity, rng)
```

Comment:
Don't test networks include the test entity?

Reply:
yeah agreed and makes sense. Maybe it would be nicer if I do that in a separate PR though as it will also affect other workloads (at least the transfer one), and I can for now just keep Fees to zero here.

Reply:
Yeah definitely in a separate PR. Could use the test entity for this yeah :+1:

Reply:
:+1: